msnbase2.Rnw

```diff
@@ -309,10 +309,21 @@ direct benefits using the on-disk backend, namely faster reading and
 reduced memory footprint. Figure \ref{fig:bench} shows 5-fold faster
 reading times (a) and over a 10-fold reduction in memory usage (b).
 
-The on-disk backend also offers efficient data manipulation by way of
+\color{red}{Because the on-disk backend does not hold all the data in memory,
+  direct data manipulations are not possible. We thus implemented a \textit{lazy
+  processing} mechanism in this backend that caches any data manipulation
```

Comment:
This is more or less the same sentence twice:
> We thus implemented a \textit{lazy processing} mechanism in this backend that caches any data manipulation
...
Operations on the \sout{raw}\color{red}{MS} data are stored} in a processing queue and only effectively
\sout{applied}\color{red}{applies them} when the user accesses m/z or intensity values

Reply:
Just a small change but what about:
We thus implemented a \textit{lazy processing} mechanism in this backend that caches any data manipulation operations in a processing queue in the object itself. Only when the user accesses m/z or intensity values these operations are effectively applied.

Reply:
Sure, sounds better. I will change